### PR TITLE
Setup UI, Configure, and make it running

### PR DESCRIPTION
**Pull request by SagarVishal**

### CODE REVIEW

File: web/.env.docker [hunk 1]
```diff
@@ -0,0 +1,3 @@
+MODE="docker"
+BASE_URL="/"
+API_URL="http://hydra:4445"
```
Comment (ekta-kakadia) [inline web/.env.docker]: add extra line at last

File: web/.env.example [hunk 2]
```diff
@@ -0,0 +1,6 @@
+MODE=development
+PORT=3000
+BASE_URL=http://localhost:3000
+
+PROXY_URL=http://pulsar.staging.pandio.com
+HYDRA_PROXY_URL=http://localhost:4445
```
Comment (ekta-kakadia) [inline web/.env.example:6]: here too

File: web/.env.staging [hunk 3]
```diff
@@ -0,0 +1,3 @@
+MODE="staging"
+BASE_URL="/"
+API_URL="http://hydra:4445"
```
Comment (ekta-kakadia) [inline web/.env.staging]: here also

Comment (SagarVishal) [reply web/.env.staging]: @ekta-kakadia Done

File: web/.env.docker (hunk 1 above)
Comment (SagarVishal) [reply web/.env.docker]: @ekta-kakadia done

File: web/.env.example (hunk 2 above)
Comment (SagarVishal) [reply web/.env.example:6]: @ekta-kakadia Done

**Review by ekta-kakadia — APPROVED** [review]

File: web/assets/css/app.css [hunk 4]
```diff
@@ -0,0 +1,127 @@
+@import "https://fonts.googleapis.com/icon?family=Material+Icons";
```
Comment (tapanbavaliya) [inline web/assets/css/app.css]: @Vishalsag remove this file

Comment (SagarVishal) [reply web/assets/css/app.css]: removed @tapanbavaliya

**Pull request merged**